samples/GetUserProfile.py

```diff
@@ -58,7 +38,7 @@
 if len(sys.argv) > 4:
     userid = sys.argv[4]
 else:
-    userid = get_input("UserID: ")
```

Comment:
I probably would have imported as `from samples_common import get_input, get_user_details` since it would require less changes to the code while still being clear. It's entirely subjective though and up to the author to decide what to import :)